mmedit/models/editors/nafnet/utils.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from torch import nn as nn
+
+
+class LayerNormFunction(torch.autograd.Function):
```

Comment:
Is it different from mn.LayerNorm?

Reply:
Yes. I preliminarily compared the behavior of this layer norm with nn.LayerNorm, nn.BatchNorm2d, nn.InstanceNorm2d. The results are different. From the code, it is a little bit similar to BatchNorm2d but with separate weights and biases.